python/whylogs/core/dataset_profile.py

```diff
@@ -82,6 +88,18 @@ def add_metric(self, col_name: str, metric: Metric) -> None:
             raise ValueError(f"{col_name} is not a column in the dataset profile")
         self._columns[col_name].add_metric(metric)
 
+    def add_dataset_metric(self, name: str, metric: Metric) -> None:
+        self._metrics[name] = metric
+
+    def add_model_performance_metric(self, metric: ModelPerformanceMetrics) -> None:
```

Comment:
Super minor nit, can be skipped: I don't quite understand why this is singular and the `get` method for model_performance_metrics is pluralized. There doesn't seem to be a way to have multiple. Should we always call it `model_performance_metrics`? Or simply `model_performance`?

Reply:
The v0 class is largely inherited here which was pluralized, so I'd say it should be pluralized consistently.

The v1 future API to add more flexible metrics with `add_metric` I think it ok.

Reply:
fixed.